base/file.jl

```diff
@@ -369,10 +369,11 @@ function cp(src::AbstractString, dst::AbstractString; force::Bool=false,
 end
 
 """
-    mv(src::AbstractString, dst::AbstractString; force::Bool=false)
+    mv(src::AbstractString, dst::AbstractString; into::Bool=false, force::Bool=false)
```

Comment:
Maybe `destdir` instead of `into`?  Don't we usually use a `dir` argument name for directories?

Reply:
The argument isn't a destination directory though which is what that argument name implies to me; it controls whether `src` is moved "onto" the `dst` path or "into" the `dst` path.